packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/DefaultStyleValuesUtil.kt

```diff
@@ -9,7 +9,6 @@ package com.facebook.react.views.text
 
 import android.content.Context
 import android.content.res.ColorStateList
-import androidx.core.content.res.use
```

Comment:
Hey @mateoguzmana just a small heads up that this triggered a lot of crashes internally. Once we remove this import, we fallback to use Kotlin's `use` (https://kotlinlang.org/api/core/kotlin-stdlib/kotlin.io/use.html) which is different than this one: https://developer.android.com/reference/kotlin/androidx/core/content/res/package-summary#(android.content.res.TypedArray).use(kotlin.Function1)

Specifically the app were crashing with `.IncompatibleClassChangeError: Class 'android.content.res.TypedArray' does not implement interface 'java.lang.AutoCloseable'`

Reply:
> Not sure if there is something we can do to ensure this regression doesn't happen again, but if there is, I'm happy to help with it

As a bare minimun having a comment on top of the import would help. Otherwise we need to suppress the IDE warning with a @Suppress annotation.

It's a bit tricky for me because I don't see a warning on that import with Android Studio so I'm a bit unsure how to suppress it